compiler/src/iree/compiler/Codegen/TransformStrategies/GPU/Strategies.h

```diff
@@ -26,6 +26,18 @@ class StagedReductionStrategy;
 static constexpr int64_t kCudaWarpSize = 32;
 static constexpr int64_t kCudaMaxNumThreads = 1024;
 
+/// Placeholder for representing supported WMMA/Cooperative Matrix
+/// configurations. This is a reflection of
+/// SPIRV_CooperativeMatrixPropertiesNVArrayAttr.
```

Comment:
SPIRV_CooperativeMatrixPropertiesNVAttr you mean? This is not an array.

Reply:
Good catch!